[{"ContainerToolsException.Message":"Las herramientas de contenedor de Visual Studio requieren que Docker se esté ejecutando antes de compilar, depurar o ejecutar un proyecto con contenedores.  Para obtener más información, consulte https://aka.ms/DockerToolsTroubleshooting","ContainerToolsException.ErrorCode":"","ContainerToolsException.StackTrace":"","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.InnerException":"Microsoft.VisualStudio.Containers.Tools.Common.ContainerToolsException: error during connect: this error may indicate that the docker daemon is not running: Get \"http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/containers/json?filters=%7B%22label%22%3A%7B%22com.docker.compose.config-hash%22%3Atrue%2C%22com.docker.compose.project%3Ddockercompose10766449611526478549%22%3Atrue%7D%7D\": open //./pipe/docker_engine: The system cannot find the file specified.\r\n   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   en Microsoft.Docker.DockerComposeClient.<ExecuteAsync>d__16.MoveNext()\r\n--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---\r\n   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   en Microsoft.Docker.DockerComposeClient.<DownAsync>d__9.MoveNext()\r\n--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---\r\n   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   en Microsoft.Docker.DockerWorkspace.<ComposeDownContainersAsync>d__34.MoveNext()\r\n--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---\r\n   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   en Microsoft.Docker.BuildTasks.ComposeDownContainers.<ExecuteCoreAsync>d__4.MoveNext()\r\n--- Fin del seguimiento de la pila de la ubicación anterior donde se produjo la excepción ---\r\n   en System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   en Microsoft.Docker.BuildTasks.DockerBuildTask.<ExecuteAsync>d__38.MoveNext()\r\nCódigo de error: DT1001\r\nGravedad: Error\r\nLínea de comandos: docker-compose  -f \"C:\\Users\\luis1\\Documents\\htdocs\\CoreJWTAngular\\docker-compose.yml\" -f \"C:\\Users\\luis1\\Documents\\htdocs\\CoreJWTAngular\\docker-compose.override.yml\" -f \"C:\\Users\\luis1\\Documents\\htdocs\\CoreJWTAngular\\obj\\Docker\\docker-compose.vs.release.partial.g.yml\" -p dockercompose10766449611526478549 --ansi never kill\r\nCódigo de salida del comando: 1\r\nSalida del comando: \r\nError de comando: error during connect: this error may indicate that the docker daemon is not running: Get \"http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/containers/json?filters=%7B%22label%22%3A%7B%22com.docker.compose.config-hash%22%3Atrue%2C%22com.docker.compose.project%3Ddockercompose10766449611526478549%22%3Atrue%7D%7D\": open //./pipe/docker_engine: The system cannot find the file specified.\r\n"}]